packages/cli/src/ActiveWebhooks.ts

```diff
@@ -157,7 +157,7 @@ export class ActiveWebhooks {
 	 *
 	 */
 	async removeWorkflow(workflow: Workflow): Promise<boolean> {
-		const workflowId = workflow.id!.toString();
+		const workflowId = workflow.id;
```

Comment:
Can we please move all the changes around workflow ID not being optional anymore into a separate PR 🙏🏽

Reply:
These are no longer here after the master merge.